packages/wallets/src/evm/chains.ts

```diff
@@ -0,0 +1,34 @@
+import { BlockchainIncludingTestnet as Blockchain, objectValues, type ObjectValues } from "@crossmint/common-sdk-base";
+
+const SmartWalletTestnet = {
+    BASE_SEPOLIA: Blockchain.BASE_SEPOLIA,
+    POLYGON_AMOY: Blockchain.POLYGON_AMOY,
+    OPTIMISM_SEPOLIA: Blockchain.OPTIMISM_SEPOLIA,
+    ARBITRUM_SEPOLIA: Blockchain.ARBITRUM_SEPOLIA,
+} as const;
+type SmartWalletTestnet = ObjectValues<typeof SmartWalletTestnet>;
+const SMART_WALLET_TESTNETS: readonly SmartWalletChain[] = objectValues(SmartWalletTestnet);
+
+const SmartWalletMainnet = {
+    BASE: Blockchain.BASE,
+    POLYGON: Blockchain.POLYGON,
+    OPTIMISM: Blockchain.OPTIMISM,
+    ARBITRUM: Blockchain.ARBITRUM,
+} as const;
+type SmartWalletMainnet = ObjectValues<typeof SmartWalletMainnet>;
+const SMART_WALLET_MAINNETS: readonly SmartWalletChain[] = objectValues(SmartWalletMainnet);
```

Comment:
i think i would just define this as like `SMART_WALLET_SUPPORTED_TESTNETS = [Blockchain.BASE_SEPOLIA, ...]`

rather than an object and having to re-define all the names of the blockchains